src/sys/pthread.rs

```diff
@@ -36,3 +38,22 @@ pub fn pthread_kill<T: Into<Option<Signal>>>(thread: Pthread, signal: T) -> Resu
     let res = unsafe { libc::pthread_kill(thread, sig) };
     Errno::result(res).map(drop)
 }
+
+/// Obtain the name of the thread
+///
+/// On linux the name cannot exceed 16 length including null terminator
+#[cfg(all(target_os = "linux", not(target_env = "musl")))]
```

Comment:
NetBSD and FreeBSD also support these functions.  You should include them, though first you'll have to add FreeBSD's to libc.

Reply:
I know but I wanted to see first it would appeal ;-) I added missing calls earlier anyway.